core/src/main/java/com/elefana/indices/fieldstats/LoadUnloadManager.java

```diff
@@ -49,12 +49,13 @@
     private Map<String, Long> lastIndexUse = new ConcurrentHashMap<>();
     private ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
 
-    public LoadUnloadManager(JdbcTemplate jdbcTemplate, State state) {
+    public LoadUnloadManager(JdbcTemplate jdbcTemplate, State state, long outdatedMinutes) {
         this.jdbcTemplate = jdbcTemplate;
         this.state = state;
+        this.OUTDATED_INDEX = outdatedMinutes * 60 * 1000;
 
         missingIndices.addAll(jdbcTemplate.queryForList("SELECT _indexname FROM elefana_field_stats_index", String.class));
-        scheduledExecutorService.scheduleAtFixedRate(this::unloadUnusedIndices, 0L, 5L, TimeUnit.MINUTES);
+        scheduledExecutorService.scheduleAtFixedRate(this::unloadUnusedIndices, 0L, outdatedMinutes / 2, TimeUnit.MINUTES);
```

Comment:
Why divide by 2?

Reply:
When the "cleaning" task is run at the same rate indices get outdated, they (indices) can stick around nearly twice as long as their time to live is.